libcalico-go/lib/backend/model/resource.go

```diff
@@ -148,6 +149,11 @@ func init() {
 		apiv3.KindKubeControllersConfiguration,
 		"kubecontrollersconfigurations",
 		reflect.TypeOf(apiv3.KubeControllersConfiguration{}))
+	registerResourceInfo(
+		KindK8sService,
+		"k8s-service",
```

Comment:
Should be "kubernetesservices" to match the other Kubernetes resources we pass through (annoying, but consistent)